src/rewards/veterans.rs

```diff
@@ -0,0 +1,35 @@
+use serde::Deserialize;
+
+pub type VeteranAdvisorId = String;
+pub type VeteranAdvisorReward = rust_decimal::Decimal;
+
+pub type VeteranAdvisorRewards = Vec<(VeteranAdvisorId, VeteranAdvisorReward)>;
+
+#[derive(Deserialize)]
+pub struct VeteranReview {
+    name: VeteranAdvisorId,
+    #[serde(alias = "No. of Reviews")]
+    number_of_reviews: usize,
+}
+
+pub fn calculate_veteran_advisors_rewards(
+    veteran_reviews: &[VeteranReview],
+    base_rewards: VeteranAdvisorReward,
+) -> VeteranAdvisorRewards {
+    let total_reviews: VeteranAdvisorReward = VeteranAdvisorReward::from(
```

Comment:
nit: type and variable names seems to disagree on what the data represents (also the type specification is redundant). I would leave this as `usize` and do the conversion later for better readability